spec/library_spec.rb

```diff
@@ -0,0 +1,51 @@
+require './lib/library.rb'
+require './lib/person.rb'
+describe Library do
+  person = Person.new("Carissa")
+  subject { described_class.new()}
+  it 'should load books from a file' do
+    expect(subject.books.length).to eq 5
```

Comment:
I'd probably just check that the `books` object is a `collection` here (`Array`, etc..). What if you change the content of the file you're reading from? Then you'd have to update this test too.